SimPEG/EM/TDEM/SrcTDEM.py

```diff
@@ -311,36 +427,29 @@ def s_e(self, prob, time):
         b = self._bSrc(prob)
 
         if prob._formulation == 'EB':
-
-            MfMui = prob.MfMui
+            MfMui = prob.mesh.getFaceInnerProduct(1./self.mu)
```

Comment:
@sgkang: this was the problem

Reply:
`prob.MfMui` should take `mu`, but to calculate the source, we need to work with `mu_0` since this is what is used to calculate the vector potential `a`. Does that make sense?